pegasus/emails/dashboard.html

```diff
@@ -0,0 +1,7 @@
+---
+from: '"Code.org" <noreply@code.org>'
+subject: "<%= subject %>"
+---
+<%= body %>
+
+<img src="<%= tracking_pixel %>"/>
```

Comment:
verified: unsubscribe does **not** apply to any messages we try to send (dashboard, or pegasus). unsubscribe is checked when mailing lists are generated and the unsubscribers are excluded from the lists.